Turbulence_PrognosticTKE.pyx

```diff
@@ -535,10 +535,13 @@ cdef class EDMF_PrognosticTKE(ParameterizationBase):
         self.update_GMV_ED(GMV, Case, TS)
         self.compute_covariance(GMV, Case, TS)
 
-        # update grid-mean cloud fraction
+        # update grid-mean cloud fraction and cloud cover
         for k in xrange(self.Gr.nzg):
+            self.EnvVar.Area.values[k] = 1.0 - self.UpdVar.Area.bulkvalues[k]
```

Comment:
I don't think an environmental variable should be updated here. We can just use it or use (1-...), but not update it.

Reply:
Discussed and agreed that it should be here. Otherwise Soares does not work with the no-clipping version.